tools/protobuf-compiler/src/main.rs

```diff
@@ -206,7 +204,8 @@ fn get_buf_from_env() -> PathBuf {
         "linux" => "You can download it from https://github.com/bufbuild/buf/releases; if you are on Arch Linux, install it from the AUR with `rua install buf` or another helper",
         _other =>  "Check if there is a precompiled version for your OS at https://github.com/bufbuild/buf/releases"
     };
-    let error_msg = "Could not find `buf` installation and this build crate cannot proceed without
+    let error_msg = "Could not find `buf` installation and this build crate cannot proceed \
+                     without
     this knowledge. If `buf` is installed and this crate had trouble finding
     it, you can set the `BUF` environment variable with the specific path to your
     installed `buf` binary.";
```

Comment:
Would be good to tidy up this string.

Reply:
Done in [6d0092d](https://github.com/astriaorg/astria/pull/1825/commits/6d0092d8130dea8717f5f924b729ea25ef8e634c)